roycemoses/cs179n-project
# Aavit004/inventory continued

To Test:
Scene to run: Sprint 6
Press I to show the Inventory and see what you currently have. 
Walk over potions and see their number increase in inventory
Press on the potion and click equip to see the number of potions decrease. When you are out the inventory slot should disappear.
